xbmc/guilib/GUIWindowManager.cpp

```diff
@@ -1174,6 +1175,9 @@ void CGUIWindowManager::FrameMove()
     m_deleteWindows.clear();
   }
 
+  if (CDataCacheCore::GetInstance().PlayStateChanged())
```

Comment:
CApplicationPlayer::FrameMove is a much better place to check this. That method also knows if a player is active